managed-os/build.f/particles-managed-os/particles/partials/cf_init.hbs

```diff
@@ -65,26 +65,15 @@
     "datadogSignatureCheck": {
         "commands": {
              "agent signature check": {
-                "command": { "Fn::Join" : ["", ["powershell.exe   $getAuthSig = Get-AuthenticodeSignature 'C:\\sungardas\\ddagent-cli-latest.msi' ; If($getAuthSig.Status -ne 'Valid') {throw $getAuthSig.path }"]] },
+                "command": {"Fn::If" : [ "isDatadogApiKeyEmpty", {"Fn::Join" : ["echo 'No Datadog APIKEY'", ["powershell.exe   $getAuthSig = Get-AuthenticodeSignature 'C:\\sungardas\\ddagent-cli-latest.msi' ; If($getAuthSig.Status -ne 'Valid') {throw $getAuthSig.path }"]]},""] },
                 "waitAfterCompletion": 0
              }
          }
     },
     "datadog": {
         "commands": {
              "install datadog": {
-                "command": {
-                  "Fn::Join": [
-                    "",
-                    [
-                      "msiexec /qn /i C:\\sungardas\\ddagent-cli-latest.msi APIKEY='",
-                      {
-                        "Ref": "datadogApiKey"
-                      },
-		      "' "
-                    ]
-                  ]
-                },
+                "command": {"Fn::If" : [ "isDatadogApiKeyEmpty", "echo 'No Datadog APIKEY'", { "Fn::Join": ["",["msiexec /qn /i C:\\sungardas\\ddagent-cli-latest.msi APIKEY='",{"Ref": "datadogApiKey"},"' "]]}]},
```

Comment:
This condition shouldnt echo No Datadog APIKEY, API Key is been passed here.

Reply:
Just debug purpose i have put echo